Source/WebKit/UIProcess/Cocoa/ProcessAssertionCocoa.mm

```diff
@@ -392,9 +392,9 @@ static ASCIILiteral runningBoardDomainForAssertionType(ProcessAssertionType asse
             });
         };
         AssertionCapability capability { process.environmentIdentifier(), runningBoardDomain, runningBoardAssertionName, WTFMove(willInvalidateBlock), WTFMove(didInvalidateBlock) };
-        m_capability = capability.platformCapability();
+        m_capability = WTFMove(capability);
```

Comment:
I would have done this:
```
m_capability  = AssertionCapability { process.environmentIdentifier(), runningBoardDomain, runningBoardAssertionName, WTFMove(willInvalidateBlock), WTFMove(didInvalidateBlock) };
```

Reply:
Ah, good point, will fix!

Thanks for reviewing!